Changelog

```diff
@@ -10,6 +10,7 @@ NOT BE PRESENT IN THE STABLE BRANCH
 For details of when a commit makes it into the stable branch see the ReleaseNotes file.
 
 2018-07-31 DariusIII
+	Chg: Update coding style of classes, third batch
```

Comment:
new addition should be added after precious ones, so they reflect the order of changes